boards/arm/google_twinkie_v2/google_twinkie_v2.yaml

```diff
@@ -0,0 +1,14 @@
+identifier: google_twinkie_v2
+name: Google Twinkie V2
+type: mcu
+arch: arm
+toolchain:
+  - zephyr
+  - gnuarmemb
+  - xtools
+ram: 36
```

Comment:
The RAM/flash  size look too small, shouldn't they be 144k/512k?

Reply:
Yes, It has now been corrected.